repos/spack_repo/builtin/packages/flecsi/package.py

```diff
@@ -18,7 +18,7 @@ class Flecsi(CMakePackage, CudaPackage, ROCmPackage):
     mesh topology, mesh geometry, and mesh adjacency information.
     """
 
-    homepage = "http://flecsi.org/"
+    homepage = "https://flecsi.org/"
```

Comment:
```suggestion
    homepage = "https://www.flecsi.org/"
```
for consistency with our own repository.